### PR TITLE
Feature 13

### PR DESCRIPTION
**Pull request by maksimmurza**
Setup front-end repository

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Ignore artifacts:
+build
+coverage
+node_modules
+.vscode
+public
+
+*.md
+.env
+.gitignore
```
Comment (slavasgod) [inline .prettierignore:10]: Add a new line here

File: src/components/App.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React, { ReactElement } from 'react';
+
+const App: React.FC = (): ReactElement => {
+	return (
+		<>
+			<span>init</span>
+		</>
+	);
+};
```
Comment (slavasgod) [inline src/components/App.tsx:9]: Lets use `tabWidth: 2` formatting. Cause in this case when you will have more nested elements your code will look weird)

File: package.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+		"@types/react-redux": "^7.1.18",
+		"axios": "^0.21.4",
+		"eslint-config-prettier": "^8.3.0",
+		"react": "^17.0.2",
+		"react-dom": "^17.0.2",
+		"react-redux": "^7.2.5",
+		"react-router-dom": "^5.3.0",
+		"react-scripts": "4.0.3",
+		"styled-components": "^5.3.1",
+		"typescript": "^4.1.6"
```
Comment (slavasgod) [inline package.json:24]: Some of the dependencies can be migrated to devDependencies (e.x. eslint-config-prettier, typescript)

**Review by slavasgod — CHANGES_REQUESTED** [review]

File: src/components/App.tsx (hunk 2 above)
Comment (maksimmurza) [reply src/components/App.tsx:9]: Should I use `tabWidth: 2` for *.html files only or for all project ?

File: .prettierignore (hunk 1 above)
Comment (maksimmurza) [reply .prettierignore]: Done

File: package.json (hunk 3 above)
Comment (maksimmurza) [reply package.json]: Prettier, eslint-config-prettier, typescript were moved to dev

File: src/components/App.tsx (hunk 2 above)
Comment (slavasgod) [reply src/components/App.tsx:9]: Yes, but you don't change every file with this formatting)

Comment (maksimmurza) [reply src/components/App.tsx]: finally...

**Review by slavasgod — APPROVED** [review]

**Pull request merged**